template/API/Hasura-GraphQL/README.md

```
@@ -0,0 +1,67 @@
+
+## 2. Apply schema and metadata migration
+
+Make sure to have hasura cli installed, check this [link](https://hasura.io/docs/2.0/hasura-cli/install-hasura-cli/)
+
+```sh
+cd hasura-exports
+hasura migrate apply --endpoint http://localhost:8080 --admin-secret password --database-name postgres
+hasura metadata apply --endpoint http://localhost:8080 --admin-secret password --database-name postgres
+```
+
+All tables and relationships can tracked also from the console (http://localhost:8080/console/data/postgres/schema/public)
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)